examples/cpp/Example_5_3_RiftRendered.cpp

```diff
@@ -10,8 +10,7 @@ class CubeScene_Rift: public RiftGlfwApp {
   PerEyeArg eyes[2];
   ovrTexture eyeTextures[2];
 
-  float ipd{ OVR_DEFAULT_IPD };
-  float eyeHeight{ OVR_DEFAULT_EYE_HEIGHT };
+  float ipd, eyeHeight;
```

Comment:
The defaults are useless (and therefore distracting.)  The member fields
are assigned values immediately in the constructor.

On Sat, Feb 28, 2015 at 7:38 PM, Brad Davis <notifications@github.com>
wrote:

> In examples/cpp/Example_5_3_RiftRendered.cpp
> <https://github.com/OculusRiftInAction/OculusRiftInAction/pull/72#discussion_r25560462>
> :
>
> > @@ -10,8 +10,7 @@ class CubeScene_Rift: public RiftGlfwApp {
> >    PerEyeArg eyes[2];
> >    ovrTexture eyeTextures[2];
> >
> > -  float ipd{ OVR_DEFAULT_IPD };
> > -  float eyeHeight{ OVR_DEFAULT_EYE_HEIGHT };
> > +  float ipd, eyeHeight;
>
> what's the purpose of removing the defaults here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OculusRiftInAction/OculusRiftInAction/pull/72/files#r25560462>
> .
>